cmd/tailscaled/install_darwin.go

```diff
@@ -43,6 +43,12 @@ const darwinLaunchdPlist = `
   <key>RunAtLoad</key>
   <true/>
 
+  <key>StandardOutPath</key>
+  <string>/var/log/tailscaled.log</string>
+
+  <key>StandardErrorPath</key>
+  <string>/var/log/tailscaled.log</string>
+
```

Comment:
Do these get rotated?

Is it okay/defined for stdout+stderr to have the same value?

Reply:
Currently the log size is unlimited, a feasible solution is to increase the newsyslog configuration, such as `/etc/newsyslog.d/wifi.conf`.

If the log is not configured, it seems that the log will be discarded and cannot be viewed.